core/modules/views/templates/views.theme.inc

```diff
@@ -162,7 +162,9 @@ function template_preprocess_views_view(&$variables) {
 }
 
 /**
- * Preprocess theme function to print a single record from a row, with fields
+ * Preprocess theme function to print a single record from a row, with fields.
+ * This hook will only be run on views whose Format settings choose to show fields.
```

Comment:
Short description and long description must be separated by an empty line.

Reply:
`This hook will only be run on views whose Format settings choose to show fields` is not quite accurate. The `Table` Display format does use fields; however, this hook will not run when using `Table`. 

<img width="650" alt="Screen Shot 2024-10-06 at 8 50 35 AM" src="https://github.com/user-attachments/assets/c1658d4f-05aa-4ed5-9b45-cccd80c5350f">

Suggestion:
`This hook will be invoked for display formats (other than Table) whose format settings choose to show fields`

Reply:
> And ideally there should be a reference to the template like:
> 
> ```
> @see views-view-field.tpl.php
> ```

Shouldn't that be 'views-view-fields.tpl.php' since this is template_preprocess_views_view_fields?

Reply:
> Shouldn't that be 'views-view-fields.tpl.php' since this is template_preprocess_views_view_fields?

Oops, you're of course right - should be ...fieldS. :+1: You already fixed it, resolved for me. :smiley: